visualization/app/codeCharta/ui/metricChooser/metricChooser.component.spec.ts

```diff
@@ -56,7 +56,7 @@ describe("metricChooserComponent", () => {
 		})
 
 		await userEvent.click(await screen.findByText("aMetric (1)"))
-		await screen.findByText("search metric (max value)")
+		await screen.getByPlaceholderText("search metric (max value)")
```

Comment:
I haven't tested it, but `screen.get*` methods are not async, in contrast to `screen.find*`
```suggestion
		screen.getByPlaceholderText("search metric (max value)")
```

Reply:
It works without await too 👍